mod/metrics/set/abstract/value/edit.rb

```diff
@@ -28,7 +28,17 @@ def edit_fields?
   end
 
   def value_field_card_and_options
-    [card, { title: "Answer", editor: :standard, show: :help }]
+    val_card =
+      if card.relationship?
+        # For adding new relationship answers.
+        # The card is the (existing) metric answer card M+C+Y+values with
+        # the company count.
+        # With this change we get the right editor for creating a new relationship value.
+        Card.new name: card.name, type_id: card.metric_card.value_cardtype_id
```

Comment:
This did the trick: Replace the card object with a new card with the same name. But it needs a decko change to make the form prefixes work.

Reply:
says 330 files changed in decko.  I'll wait for the decko PR :)

Reply:
I assume so :).  You can back out or we can push ahead with the bridge, either way.